### ISSUE TITLE
Present to the team (Monday)

### ISSUE THREAD

**Issue by fame-odoo** [fame-odoo/OdooSalesDashboard#4]

**Comment by kusn-odoo** [comment]
Given the list of features. We should ask for the most important features so that we can prioritize features.

**Comment by kusn-odoo** [comment]
> > Given the list of features. We should ask for the most important features so that we can prioritize features.
> 
> Maybe we should make a list here?

Yes, we should  definitely make a list to keep track of the features during development

**Comment by kusn-odoo** [comment]
[salesdashboard.pdf](https://github.com/fame-odoo/OdooSalesDashboard/files/9084711/salesdashboard.pdf)

@fame-odoo Here's the blueprint for the sales dashboard. please let me know if anything can be added

**Comment by MFYDev** [comment]
![Overview-Sales](https://user-images.githubusercontent.com/53750381/178386040-6d0f162b-4d1b-40f4-a48e-78a5d2f1e38c.png)

**Comment by MFYDev** [comment]
![Salesperson-Report](https://user-images.githubusercontent.com/53750381/178406970-3d6eb0f7-8dcc-4579-8cb4-d8c1863914eb.png)

**Comment by MFYDev** [comment]
Things need to be adjusted:

1. Show top 10 salesperson of both offices
    - Add which office do they come from in the last column

2. Make the default view as the current month
    - Automatically pull the data and refresh the page daily
